inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convolution.hpp

```diff
@@ -17,23 +17,24 @@ namespace LayerTestsDefinitions {
 
 // ! [test_convolution:definition]
 typedef std::tuple<
-        InferenceEngine::SizeVector,    // Kernel size
-        InferenceEngine::SizeVector,    // Strides
-        std::vector<ptrdiff_t>,         // Pad begin
-        std::vector<ptrdiff_t>,         // Pad end
-        InferenceEngine::SizeVector,    // Dilation
-        size_t,                         // Num out channels
-        ngraph::op::PadType             // Padding type
+        InferenceEngine::SizeVector,                         // Kernel size
+        InferenceEngine::SizeVector,                         // Strides
+        std::vector<ptrdiff_t>,                              // Pad begin
+        std::vector<ptrdiff_t>,                              // Pad end
+        InferenceEngine::SizeVector,                         // Dilation
+        size_t,                                              // Num out channels
+        ngraph::op::PadType                                  // Padding type
 > convSpecificParams;
 typedef std::tuple<
         convSpecificParams,
-        InferenceEngine::Precision,     // Net precision
-        InferenceEngine::Precision,     // Input precision
-        InferenceEngine::Precision,     // Output precision
-        InferenceEngine::Layout,        // Input layout
-        InferenceEngine::Layout,        // Output layout
-        InferenceEngine::SizeVector,    // Input shapes
-        LayerTestsUtils::TargetDevice   // Device name
+        InferenceEngine::Precision,                          // Net precision
+        InferenceEngine::Precision,                          // Input precision
+        InferenceEngine::Precision,                          // Output precision
+        InferenceEngine::Layout,                             // Input layout
+        InferenceEngine::Layout,                             // Output layout
+        std::vector<std::vector<std::pair<size_t, size_t>>>, // Input shape
```

Comment:
Not necessary to use such extra big constriction. The convolution has only one input.
In the previous version we used `InferenceEngine::SizeVector`. It is confusing again

Reply:
This is planned to support eltwise style input as well.

Reply:
As for softmax, it is applicable for multiinput nodes but softmax is one input node